magma_smaht/wrangler_utils.py

```diff
@@ -125,3 +125,20 @@ def reset_all_failed_mwfrs(smaht_key: dict):
 def print_error_and_exit(error):
     print(error)
     exit()
+
+
+def set_property(
+    uuid: str,
+    prop_key: str,
+    prop_value: Union[str,int],
+    smaht_key: Dict[str, Any]
+    ):
+    """"Sets a property prop_key to value prop_value for item with uuid."""
+    patch_body={
+        prop_key: prop_value
+    }
+    try:
+        ff_utils.patch_metadata(patch_body, obj_id=uuid, key=smaht_key)
+        print(f"Set item {uuid} property {prop_key} to {prop_value}.")
+    except Exception as e:
+        raise Exception(f"Item could not be PATCHed: {e}")
```

Comment:
We might need `str(e)` here. This might be wrong in other places. Please correct those as well.